java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -3723,6 +3724,20 @@ public T lmpop(@NonNull String[] keys, @NonNull PopDirection direction) {
         return getThis();
     }
 
+    /**
+     * * Sets the list element at <code>index</code> to <code>element</code>.
```

Comment:
```suggestion
     * Sets the list element at <code>index</code> to <code>element</code>.
```
and copy doc for `index`